xtext-gradle-plugin/src/main/java/org/xtext/gradle/tasks/XtextGenerate.xtend

```diff
@@ -151,6 +151,14 @@ class XtextGenerate extends DefaultTask {
 			initializeBuilder
 		}
 		val request = new GradleInstallDebugInfoRequest => [
+			//Issue #37 Fix
```

Comment:
this comment is not needed, just include the issue number in your commit message